ray_lightning/launchers/ray_horovod_launcher.py

```diff
@@ -0,0 +1,251 @@
+from typing import Callable, Any, Optional
```

Comment:
All the comments in this file apply to ray_launcher as well

Reply:
sounds good to me